knack/invocation.py

```diff
@@ -137,7 +137,7 @@ def execute(self, args):
         self.parser.load_command_table(self.commands_loader)
         self.cli_ctx.raise_event(EVENT_INVOKER_CMD_TBL_LOADED, parser=self.parser)
 
-        arg_check = [a for a in args if a not in ['--verbose', '--debug', '--only-show-warnings']]
```

Comment:
It's overwritten in azure-cli-core

https://github.com/Azure/azure-cli/blob/dce83e308546a3fda4085055b35d7b69fff43b33/src/azure-cli-core/azure/cli/core/commands/__init__.py#L565